frontend/public/components/service-catalog/create-instance.tsx

```diff
@@ -147,43 +144,45 @@ class CreateInstance extends React.Component<CreateInstanceProps, CreateInstance
       <Helmet>
         <title>{title}</title>
       </Helmet>
-      <PageHeading title={title} />
-      <div className="co-m-pane__body co-create-service-instance">
-        <div className="row">
-          <div className="col-md-7 col-md-push-5 co-catalog-item-info">
-            <ClusterServiceClassInfo obj={serviceClass} />
-          </div>
-          <div className="col-md-5 col-md-pull-7">
-            <form className="co-create-service-instance">
-              <div className="form-group co-create-service-instance__namespace">
-                <label className="control-label co-required" htmlFor="dropdown-selectbox">Namespace</label>
-                <NsDropdown selectedKey={this.state.namespace} onChange={this.onNamespaceChange} id="dropdown-selectbox" />
-              </div>
-              <div className="form-group co-create-service-instance__name">
-                <label className="control-label co-required" htmlFor="name">Service Instance Name</label>
-                <input className="form-control"
-                  type="text"
-                  onChange={this.onNameChange}
-                  value={this.state.name}
-                  id="name"
-                  required />
-              </div>
-              <div className="form-group co-create-service-instance__plans">
-                <label className="control-label">Plans</label>
-                {_.isEmpty(availablePlans)
-                  ? <p>There are no plans currently available for this service.</p>
-                  : planOptions}
-              </div>
-            </form>
-            <ServiceCatalogParametersForm schema={schema} uiSchema={uiSchema} onSubmit={this.save} formData={this.state.formData} onChange={this.onFormChange}>
-              <ButtonBar errorMessage={this.state.error} inProgress={this.state.inProgress}>
-                <button type="submit" className="btn btn-primary">Create</button>
-                <button type="button" className="btn btn-default" onClick={history.goBack}>Cancel</button>
-              </ButtonBar>
-            </ServiceCatalogParametersForm>
+      <StatusBox data={serviceClass} loaded={loaded} loadError={loadError} label="Service Class">
```

Comment:
Can we give a little more info.?  Such as what type of object was not found?  Can use the `label` in the [404](https://github.com/openshift/console/blob/master/frontend/public/components/utils/status-box.jsx#L65) response, like we do for the [`<LoadError ..>`](https://github.com/openshift/console/blob/master/frontend/public/components/utils/status-box.jsx#L79)?